cartography/intel/aws/__init__.py

```diff
@@ -29,10 +29,12 @@ def _sync_one_account(session, boto3_session, account_id, regions, sync_tag, com
     # EC2
     # TODO move this to EC2 module
     logger.info("Syncing EC2 for account '%s'.", account_id)
+    ec2.sync_vpc(session, boto3_session, account_id, sync_tag, common_job_parameters)
```

Comment:
I might not be searching correctly but I can't find this function defined in the EC2 module (https://github.com/lyft/cartography/blob/20f64d18f139d10f281e88016deb2c76438d547a/cartography/intel/aws/ec2.py), can you link it to me?